PlatformConfigurations/IEEE802.1xPortBasedAuthentication.md

```diff
@@ -0,0 +1,872 @@
+---
+title: IEEE 802.1x (Port Based Authentication)
+parent: Platform Configuration
+nav_order: 5
+layout: default
+---
+
+# IEEE 802.1x (Port Based Authentication)
+
+## Introduction
+
+In this guide, we will quickly explain what IEEE 802.1x (Port Based Authentication) is
+and give an example implementing it.
+
+IEEE 802.1x is a standard for port-based network access control that provides an authentication
+mechanism to devices wishing to join a LAN or WLAN. IEEE 802.1x uses the Extensible Authentication Protocol
+(EAP) to transmit authentication information over ethernet frameworks. The three main components
+include the **Supplicant, Authenticator, and Authentication Server**.
+
+- **Supplicant**: The Supplicant is the device attempting to access the network. It is often the one that
+  participates in the EAP transaction with the Authenticator.
+
+- **Authenticator**: The Authenticator is the network device that connects the supplicant to the network. It acts as the
+  access point for the Supplicant and controls the type of traffic flow between the Supplicant and
+  the network before and after authentication. The Authenticator sends
+  EAP requests to the Supplicant to identify a connecting device.
+  In IEEE 802.1x, this is also more broadly refered to as a Network Access Server.
+
+- **Authentication Server**: The Authentication Server receives requests for network access
+  and verifies devices attempting to connect to the network. This information is then passed back to the authenticator
+  where traffic flow is changed based on the results. This is also commonly refered to as a RADIUS Server.
+  It is also often known as an AAA (Authentication, Authorization, and Accounting) Server.
+
+The image below is a good visualization of the 802.1x Progression.
+
+![Seqeunce Diagram of the 802.1x Progression](../Images/ImagesForPlatformConfiguration/802.1x_Progression.png)
+
+In this document, we will be using the following free publicly available software:
+FreeRADIUS, hostapd, and wpa_supplicant to configure an example of IEEE 802.1x and MAC Address Authentication.
+
+## Example Configuration
+
+Consider the following topology:
+
+![IEEE-802.1x-Configuration](../Images/ImagesForPlatformConfiguration/IEEE-802.1x-ConfigurationExample.png)
+
+First, we will guide you through how to configure 802.1x and then
```

Comment:
where are you configure these? In the radius server on the dent device?

Reply:
An example configuration and explanation is provided for 3 separate devices a Supplicant, Authenticator, and RADIUS Server. I have reworded the document to clarify three devices in order to help individuals who may be attempting to configure .1x 

Clarifying 3 separate device:

![image](https://github.com/user-attachments/assets/c6b12a23-6ccf-4bee-95d5-ada966e578c9)

If you are attempting to configure a RADIUS Server a demo and explanation of key parts are provided:

![image](https://github.com/user-attachments/assets/7203d003-b69d-4077-ba0d-7a1f625317b3)

If you are attempting to configure an Authenticator the following is a demo and explanation of key parts:

![image](https://github.com/user-attachments/assets/668c0811-749d-471f-bfcf-fde8174c9484)

Lastly, the Supplicant now more clearly has a labeled section for an example configuration and explanation of key parts: 

![image](https://github.com/user-attachments/assets/21423b9d-360a-4bde-b8f5-09a8b18dba63)